README.md

```diff
@@ -422,6 +422,36 @@ the *Permanently delete previous versions of objects* checkbox, and enter *30*
 for *Number of days after objects become previous versions*. Then click *Create
 rule.*
 
+To enable CloudWatch logs for API Gateway, an IAM role must be created. This
+needs to be done once per region. Deployments with CloudWatch enabled  are
+automatically covered by this configuration.
+
+1. Navigate to the *IAM Management Console*
+
+2. Click *Roles* under *Access management*
+
+3. Click *Create role*
+
+4. Click *AWS service* for *Select type of trusted entity*
+
+5. Click *API Gateway* for *Choose a use case*, click *Next: Permissions*
+
+6. Ensure *AmazonAPIGatewayPushToCloudWatchLogs* is listed exclusively as the
+policy and click *Next: Tags*
+
+7. Add *name: azul-api_gateway* as a tag and click *Next: Review*
+
+8. For *Role name* enter `azul-api_gateway`, ensure *Policies* are as prescribed
+in step 6 and click *Create role*
+
+9. Be sure to copy the *Role ARN* for the newly created `azul-api_gateway` role
+and go to the *API Gateway* console
+
+10. Select any available Gateway resource (it doesn't matter which because this 
```

Comment:
What if the current region has no gateways? Maybe there needs to be an intro paragraph stating that this must be done after at least one deployment was created in the configured AWS account and region.

Reply:
```suggestion
10. Select any available API Gateway resource (it doesn't matter which because this 
```